org.eclipse.buildship.ui/plugin.xml

```diff
@@ -263,6 +263,50 @@
 
     <!-- properties view for tasks and projects -->
     <extension
+          point="org.eclipse.debug.ui.launchShortcuts">
+       <shortcut
+             class="org.eclipse.buildship.ui.launch.TestLaunchShortcut"
+             id="org.eclipse.buildship.ui.shortcut.test"
+             label="Gradle Test"
+             modes="run">
+          <configurationType
+                id="org.eclipse.buildship.core.launch.runconfiguration">
+          </configurationType>
+          <contextualLaunch>
+             <contextLabel
+                   label="Gradle Test"
+                   mode="run">
+             </contextLabel>
+             <enablement>
+                <with
+                      variable="selection">
+                   <iterate
+                         ifEmpty="false"
+                         operator="and">
+                      <adapt
+                            type="org.eclipse.core.resources.IResource">
+                         <test
+                               forcePluginActivation="true"
+                               property="org.eclipse.core.resources.projectNature"
+                               value="org.eclipse.buildship.core.gradleprojectnature">
+                         </test>
+                      </adapt>
+                      <adapt type="org.eclipse.jdt.core.IJavaElement">
+		                <test property="org.eclipse.jdt.core.isInJavaProject"/>
```

Comment:
As discussed today, we should not restrict the users what classes are they trying to launch as tests. Just let the users launch any classes/methods they can find.